### ISSUE TITLE
Can result.txt output format be customized? like saving as a series

### ISSUE THREAD

**Issue by kennethchanchifung** [AlexeyAB/darknet#3441]
`darknet.exe detector test cfg/coco.data cfg/yolov3.cfg yolov3.weights -dont_show -ext_output < data/train.txt > result.txt`

From the code above, I can successfully generate **one single output file** result.txt, which is great.
However, as I would like to run the yolo model in **real time**. 
Meanwhile, I would like to further analyze the output in real time (by another program), such as how many objects in the scene, the size of the object, distance between objects.
I don't think it would be a good idea if the yolo model is keep updating the result.txt file and another program keep reading and analyzing the same file.
Thus, this is why I want to customize how the output results.txt is saved.
**Let say, if the output file can be saved (such as every 10 seconds) into a NEW result.txt periodically, this can solve my problem.**

In conclusion, my objective:
How to set the output will be saved into **multiple** result.txt periodically?

Thanks so much!

**Comment by Sudhakar17** [comment]
@AlexeyAB : It will be great, if the yolov3 saves the result like the input format. so we can load the results using labeling tools.

**Comment by AlexeyAB** [comment]
@Sudhakar17 

It is implemented for detection on list of images: https://github.com/AlexeyAB/darknet#how-to-use-on-the-command-line

Pseudo-lableing - to process a list of images data/new_train.txt and save results of detection in Yolo training format for each image as label <image_name>.txt (in this way you can increase the amount of training data) use: `darknet.exe detector test cfg/coco.data cfg/yolov3.cfg yolov3.weights -thresh 0.25 -dont_show -save_labels < data/new_train.txt`

**Comment by AlexeyAB** [comment]
@kennethchanchifung For this you should implement it by yourself, by using Darknet as DLL/SO-library and edit Python/C++ example that uses this library.
